terratag_test.go

```diff
@@ -44,7 +44,11 @@ func TestTerraform14(t *testing.T) {
 }
 
 func TestTerraform15(t *testing.T) {
-	testTerraform(t, "15")
+	testTerraform(t, "15_1.0")
+}
+
+func TestTerraform1o0(t *testing.T) {
+	testTerraform(t, "15_1.0")
```

Comment:
how come the tests for 1 and 15 are the same?

Reply:
We do the same between 13 and 14.
Basically nothing has changed in terms of in/out syntax and no new cases, so we reuse the fixture files